contracts/evmscript/executors/CallsScript.sol

```diff
@@ -55,6 +58,7 @@ contract CallsScript is BaseEVMScriptExecutor {
 
             require(success, ERROR_CALL_REVERTED);
         }
+        // No need to allocate empty bytes for the return as this can only be called via an delegatecall
```

Comment:
Given the modifier checks this has, it's actually impossible to call this without going through a delegatecall.

Reply:
It may be obvious, but why the `isInitialized` would prevent a normal call?

Reply:
`isInitialized` checks against the slot at `INITIALIZATION_BLOCK_POSITION`, which is left false in this executor contract (executors are deployed as-is).

We added `isInitialized` in an attempt to make calling these scripts harder to do outside of Aragon apps (before anyone could make a call to them), so this protects against:

- Direct calls (imagine if this script changed some state; we would only want to do so if it was `delegatecall`ed into)
- Usage from uninitialized (or "petrified") Aragon apps

You could still make a contract that sets the storage slot and `delegatecall`s into this executor.

Reply:
Thanks for clarifying :)